src/Model/Resolver/Products/Query/Search.php

```diff
@@ -139,6 +157,11 @@ public function getResult(
         $searchCriteria->setPageSize($realPageSize);
         $searchCriteria->setCurrentPage($realCurrentPage);
 
+        // Following lines are added to increment search terms
```

Comment:
"Following lines are added to" are extra words which can be pre-pended to any code fragment.
In order to be concise, let's remove them!